enterprise_subsidy/apps/api/v1/views/transaction.py

```diff
@@ -65,6 +65,10 @@ class TransactionViewSet(
     lookup_field = "uuid"
     serializer_class = TransactionSerializer
     pagination_class = TransactionListPaginator
+    filter_backends = [filters.SearchFilter]
```

Comment:
[suggestion, non-blocking] There will be ordering on the `Spent` table backed by this `transactions` API. While we're in here, we might also consider adding the `filters.OrderingFilter` (unless that's slated for another ticket, ofc!)?

<img width="668" alt="image" src="https://github.com/openedx/enterprise-subsidy/assets/2828721/0e91a998-23cd-4a5c-b7fb-006ff16ec7ca">

Reply:
thats a great call out - not sure if there is an existing ticket. i'll talk to the PM